spectrum_utils/plot.py

```diff
@@ -167,22 +185,9 @@ def spectrum(
     if ax is None:
         ax = plt.gca()
 
+    _format_ax(ax, grid)
     ax.yaxis.set_major_formatter(mticker.PercentFormatter(xmax=1.0))
-    ax.set_ylim(*(0, 1) if not mirror_intensity else (-1, 0))
-
-    ax.xaxis.set_minor_locator(mticker.AutoLocator())
-    ax.yaxis.set_minor_locator(mticker.AutoLocator())
-    ax.xaxis.set_minor_locator(mticker.AutoMinorLocator())
-    ax.yaxis.set_minor_locator(mticker.AutoMinorLocator())
-    if grid in (True, "both", "major"):
-        ax.grid(True, "major", color="#9E9E9E", linewidth=0.2)
-    if grid in (True, "both", "minor"):
-        ax.grid(True, "minor", color="#9E9E9E", linewidth=0.2)
-    ax.set_axisbelow(True)
-
-    ax.tick_params(axis="both", which="both", labelsize="small")
-
-    ax.set_xlabel("m/z", style="italic")
+    ax.set_ylim(*(0, 1.1) if not mirror_intensity else (-1.1, 0))
```

Comment:
**praise:** Thanks for increasing the y-axis limit. The defaults were indeed sub-optimal and I've seen several examples in papers and during presentations that indicate that people don't modify it themselves.

Reply:
Increased to 1.15, as 1.1 was still insufficient in most cases.